packages/odyssey-react-mui/src/labs/AppSwitcher/AppSwitcherApp.tsx

```diff
@@ -0,0 +1,138 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import styled from "@emotion/styled";
+import { Skeleton } from "@mui/material";
+import { useCallback } from "react";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "../../OdysseyDesignTokensContext";
+import { Tooltip } from "../../Tooltip";
+import { MuiPropsContext, MuiPropsContextType } from "../../MuiPropsContext";
+
+const APP_SIDE_LENGTH_VAL = 36;
+const APP_SIDE_LENGTH_REM = `${APP_SIDE_LENGTH_VAL / 14}rem`;
+const APP_ICON_SIDE_LENGTH = `${32 / 14}rem`;
```

Comment:
We should probably be using token values for these. If we don't have token values for these sizes already, the sizes are likely incorrect. We'd have to go back to design and tell them we're using a token that already exists